### DISCUSSION TITLE
General Questions

### DISCUSSION THREAD

**Discussion by MarkW06071969** [MarkW06071969/Tafe202206#16] [General]
I have raised three general questions. I think we are in agreement on these but just to make sure - here they are:

1. Do all forms need to be the same size?

2. Are all forms going to be the same background colour?

3. When merging the feature branches into the development branch, should we create
    a separate merge folder on our local drive and clone the repository( las in the week 5 exercises)
    or do we do our merge work in our local tafe202206 repository? Or is either okay?

**Comment by StephenHillenbrand** [comment]
1. If the forms are using a layout that will adapt to resize this should not be an issue as the will automatically change to the parent frame.
2. I think its a good idea to keep consistency through out the project and have the backgrounds all the same colour.
3. I don't think either was is a problem, I did a git sync into my local working directory tafe202206
